config/initializers/faraday.rb

```diff
@@ -0,0 +1,7 @@
+$tmdb = Faraday.new(url: 'https://api.themoviedb.org/3/') do |faraday|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.